lstchain/reco/volume_reducer.py

```diff
@@ -92,7 +91,12 @@ def check_and_apply_volume_reduction(event, config):
 
             image[~pixels_to_keep] = 0
             pulse_time[~pixels_to_keep] = 0
-            waveform[~pixels_to_keep, :] = 0
+            if waveform.ndim == 2:
```

Comment:
do we still have cases in which we do not apply gain selection or this is just for testing purposes?

Reply:
We actually do not apply gain selection in real data currently (in the sense that we currently keep both channels).

Reply:
ok, sorry this is for R0 data

Reply:
Hum, no DL0

Reply:
yeah, I meant before we apply gain selection (for DL1)

Reply:
So if it's fine with you, I'd like to merge this